build.sbt

```diff
@@ -16,6 +16,9 @@ lazy val core = project
   .settings(framelessSettings: _*)
   .settings(warnUnusedImport: _*)
   .settings(publishSettings: _*)
+  .settings(libraryDependencies ++= Seq(
```

Comment:
Why this change?

Reply:
I forgot to take it out when I removed the `add_months` function. removed